A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux

```
CMake Error at libqmenumodel/src/CMakeLists.txt:53 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
```
CMake Error at libqmenumodel/QMenuModel/CMakeLists.txt:22 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete 

Same as https://github.com/ubports/ubuntu-download-manager/issues/4

--- a/libqmenumodel/QMenuModel/CMakeLists.txt
+++ b/libqmenumodel/QMenuModel/CMakeLists.txt
@@ -13,13 +13,14 @@ include_directories(
     ${GLIB_INCLUDE_DIRS}
 )
 
+find_package(Qt5Qml REQUIRED)
+find_package(Qt5Quick REQUIRED)
 target_link_libraries(qmenumodel-qml
     qmenumodel
     ${GLIB_LDFLAGS}
     ${GIO_LDFLAGS}
+    Qt5::Qml Qt5::Quick
 )
-
-qt5_use_modules(qmenumodel-qml Qml Quick)
 
 execute_process(COMMAND ${CMAKE_COMMAND} -E copy_if_different "${CMAKE_CURRENT_SOURCE_DIR}/qmldir"
                                                               "${CMAKE_CURRENT_BINARY_DIR}/qmldir")

--- a/libqmenumodel/src/CMakeLists.txt
+++ b/libqmenumodel/src/CMakeLists.txt
@@ -45,12 +45,12 @@ include_directories(
     ${GIO_INCLUDE_DIRS}
 )
 
+find_package(Qt5Quick REQUIRED)
 target_link_libraries(${SHAREDLIBNAME}
     ${GLIB_LDFLAGS}
     ${GIO_LDFLAGS}
+    Qt5::Core Qt5::Qml Qt5::Quick
 )
-
-qt5_use_modules(${SHAREDLIBNAME} Core Qml Quick)
 
 install(TARGETS ${SHAREDLIBNAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 

